### PR TITLE
Add Scala Examples Auto-Test II

### PR DESCRIPTION
**Pull request by respecteverything**
- [x] zoo/src/main/scala/com/intel/analytics/zoo/examples/vnni
- [x] zoo/src/main/scala/com/intel/analytics/zoo/examples/resnet

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/run-scala-example-test.sh [hunk 1]
```diff
@@ -8,6 +8,139 @@ export ANALYTICS_ZOO_HOME=$ANALYTICS_ZOO_ROOT/dist
...
+${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
+--master ${MASTER} \
+--total-executor-cores 4 \
+--class com.intel.analytics.zoo.examples.resnet.TrainImageNet \
+-f hdfs://172.168.2.181:9000/imagenet_small \
+--batchSize 8192 --nEpochs 6 --learningRate 0.1 --warmupEpoch 5 \
+--maxLr 3.2 --cache /cache  --depth 50 --classes 1000
+
+now=$(date "+%s")
+time15=$((now-start))
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/run-scala-example-test.sh]: why is time15 for # 5?

File: apps/run-scala-app-test.sh [hunk 2]
```diff
@@ -54,5 +54,195 @@ $SPARK_HOME/bin/spark-submit \
...
+./flink-1.7.2/bin/stop-cluster.sh
+
+echo "#1 fraud-detection time used:$time1 seconds"
+echo "#2 text-classification-training time used:$time2 seconds"
+echo "#3.1 text-classification-inference:SimpleDriver time used:$time3 seconds"
+echo "#3.2 text-classification-inference:WebServiceDriver time used:$time4 seconds"
+echo "#4.1 recommendation-inference:SimpleScalaDriver time used:$time5 seconds"
+echo "#4.2 recommendation-inference:SimpleDriver time used:$time6 seconds"
+echo "#5.1 model-inference-flink:Text Classification time used:$time7 seconds"
+echo "#5.2 model-inference-flink:Resnet50 Image Classification time used:$time8 seconds"
```
Comment (qiyuangong) [inline apps/run-scala-app-test.sh]: Add a blank line at the end of each file. :)

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/run-scala-example-test.sh [hunk 3]
```diff
@@ -427,14 +370,10 @@ echo "#2.3 LocalEstimator:TransferLearning used:$time4 seconds"
...
-echo "#2.1 text-classification-inference:SimpleDriver time used:$time9 seconds"
-echo "#2.2 text-classification-inference:WebServiceDriver time used:$time10 seconds"
-echo "#3.1 recommendation-inference:SimpleScalaDriver time used:$time11 seconds"
-echo "#3.2 recommendation-inference:SimpleDriver time used:$time12 seconds"
-echo "#4.1 model-inference-flink:Text Classification time used:$time13 seconds"
-echo "#4.2 model-inference-flink:Resnet50 Image Classification time used:$time14 seconds"
+echo "#5 Resnet time used:$time8 seconds"
+echo "#6.1 OpenVINO Resnet time used:$time9 seconds"
+echo "#6.2 BigDL Resnet time used:$time10 seconds"
+
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/run-scala-example-test.sh]: remove useless blank lines

**Comment by respecteverything** [comment]
Jenkins pass: http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Scala-Simple-Integration/42/console

**Comment by respecteverything** [comment]
Jenkins pass : 
http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Scala-Simple-Integration/44/console

**Comment by respecteverything** [comment]
Scala Apps Test Jenkins Pass : http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Scala-AppTests/58/

**Comment by respecteverything** [comment]
@qiyuangong @hkvision @glorysdj

**Pull request merged**